site/content/docs/0.3/helpers/dividers.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: docs
+title: Dividers
+description: Use the horizontal or vertical rule helpers to create dividers. A divider can be used to visually structure an interface by clearly separating content sections.
+group: helpers
+aliases:
+  - "/docs/helpers/dividers/"
+toc: true
+---
+
+## Horizontal rules
+
+Horizontal rules should use the `<hr>` tag, when the separator has semantic meaning (i.e., when it represents a topic change or a significant transition in the content). If the divider is purely decorative, use a `<div>` styled with a `hr` class ensuring it is hidden from assistive technologies (e.g., `aria-hidden="true"`).
+
+{{< example >}}
+<hr>
+<hr class="text-brand-primary">
+<div class="border-top border-default my-medium"></div>
+<div class="border-top border-emphasized border-thick my-medium"></div>
+{{< /example >}}
+
+
+## Vertical rules
+
+Vertical rules are inspired by the `<hr>` element, allowing you to create vertical dividers in common layouts. They're styled just like `<hr>` elements. They have `min-height` of `1em`.
```

Comment:
TBH `1em` looks a bit arbitrary, but I can't figure out why it shouldn't be this value.